lib/analysis_options.yaml

```diff
@@ -197,6 +207,8 @@ linter:
     unnecessary_string_interpolations: true # flutter
     unnecessary_this: true # flutter
     unnecessary_to_list_in_spreads: true
+    use_colored_box: false
```

Comment:
I am generally against using a container unless a large group of attributes are combined to require nesting. I'd prefer these to be true

Reply:
set to true